github/PullRequestReview.py

```diff
@@ -100,6 +100,22 @@ def delete(self) -> None:
         """
         headers, data = self._requester.requestJsonAndCheck("DELETE", f"{self.pull_request_url}/reviews/{self.id}")
 
+    def edit(self, body: str) -> None:
+        """
+        :calls: `PUT /repos/{owner}/{repo}/pulls/{number}/reviews/{review_id}
+                <https://docs.github.com/en/rest/pulls/reviews#update-a-review-for-a-pull-request>`_
+        """
+        assert isinstance(body, str), body
+        post_parameters = {
+            "body": body,
+        }
+        headers, data = self._requester.requestJsonAndCheck(
+            "PUT",
+            f"{self.pull_request_url}/reviews/{self.id}",
+            input=post_parameters,
+        )
+        self._useAttributes(data)
```

Comment:
I am wondering why we do not call:
```
        self._useAttributes(data)
```
in: `dismiss` above?

Since the `state` should change to `DISMISSED`.

Reply:
Looks like we should, can you add this here too?

Reply:
Done here: https://github.com/PyGithub/PyGithub/pull/2854